sdk/dotnet/Iam/GetWorkloadIdentityPool.cs

```diff
@@ -18,15 +18,6 @@ public static Task<GetWorkloadIdentityPoolResult> InvokeAsync(GetWorkloadIdentit
 
     public sealed class GetWorkloadIdentityPoolArgs : Pulumi.InvokeArgs
     {
-        [Input("description")]
-        public string? Description { get; set; }
-
-        [Input("disabled")]
-        public bool? Disabled { get; set; }
-
-        [Input("displayName")]
-        public string? DisplayName { get; set; }
```

Comment:
That's a breaking change code-wise - but it looks like a bug fix?

Reply:
So these values were never set and were only introduced last week in the upgrade to v3.47.0

I am going to apply the rule of thumb that this will affect < 1% of users so we are ok for this to be merged